.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+# linting check of radis-app
+name: 🌱 CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    name: 🧪 Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
```

Comment:
Might as well go ahead and upgrade this to v3 in case the caching doesn't work for this version.